10up/distributor
# fix missing css

### Description of the Change
Fixes a missing script. The admin-settings.css and admin.css had identical IDs when be enqueued, causing the admin-settings.css not to load, thus breaking the Registration Key validity indicator.


Closes #1207 

### How to test the Change

1. Visit Distributor -> Settings
2. Enter a valid or invalid registration key and click save settings
3. Observe the indicator icon next to the input displays
![image](https://github.com/10up/distributor/assets/6152801/52813f29-2af2-4c9e-94e9-e82cc41c5e00)


### Changelog Entry


Fixed - Missing admin script

### Credits

Props @psorensen 

### Checklist:
